package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@lunie/cosmos-ledger",
-  "version": "0.1.8",
-  "description": "provide simple Ledger tooling for the Cosmos Ledger App with user friendly errors",
+  "name": "@kava-labs/cosmos-ledger",
+  "version": "0.1.0-beta.0",
```

Comment:
can we update this to be like `ledgerjs`? it's way too confusing to have 2 repos: ledger-cosmos and cosmos-ledger